modules/@apostrophecms/notification/ui/public/notify.js

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
This does not belong in ui/public, notifications are for the apos view only, and the use of async/await in ui/public would break older browsers for the public view.

Instead, add this method to the apos object in your AposNotification.js "app" code. That will only run at the appropriate time (a user who is getting the rest of the code for notifications).

Reply:
Done